client/signup/steps/user/index.jsx

```diff
@@ -458,7 +458,22 @@ export class UserStep extends Component {
 	}
 
 	getHeaderText() {
-		const { flowName, oauth2Client, translate, headerText, wccomFrom, isSocialFirst } = this.props;
+		const {
+			flowName,
+			oauth2Client,
+			translate,
+			headerText,
+			wccomFrom,
+			isSocialFirst,
+			userLoggedIn,
+			locale,
+		} = this.props;
+
+		if ( userLoggedIn ) {
+			if ( englishLocales.includes( locale ) ) {
```

Comment:
Nit (obvs!): maybe just me, but two if statements feels a bit weird for the same return.